From: Philip Withnall <withnall@endlessm.com>
Date: Tue, 20 Jun 2017 17:38:15 +0100
Subject: =?utf-8?q?build=3A_Add_=E2=80=98devel=E2=80=99_or_=E2=80=98releas?=
 =?utf-8?q?e=E2=80=99_to_OSTREE=5FFEATURES_for_test-symbols=2Esh?=

test-symbols.sh was looking for the DevelBuild string, which is actually
part of the output from `ostree --version`, not $OSTREE_FEATURES.

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Closes: #948
Approved by: cgwalters

Origin: upstream, 2017.8, commit:fd50c9732231895b18cd8158726e7f77bc8dcae6
---
 configure.ac          | 1 +
 tests/test-symbols.sh | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 58795a7..65a354c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -464,6 +464,7 @@ AM_COND_IF([BUILDOPT_IS_DEVEL_BUILD],
             AC_DEFINE([BUILDOPT_IS_DEVEL_BUILD], [1], [Define if doing a development build])
             release_build_type=devel,
             release_build_type=release)
+OSTREE_FEATURES="$OSTREE_FEATURES $release_build_type"
 
 AC_CONFIG_FILES([
 Makefile
diff --git a/tests/test-symbols.sh b/tests/test-symbols.sh
index 3f62730..4a11183 100755
--- a/tests/test-symbols.sh
+++ b/tests/test-symbols.sh
@@ -22,7 +22,7 @@ set -xeuo pipefail
 echo '1..3'
 
 released_syms=${G_TEST_SRCDIR}/src/libostree/libostree-released.sym
-if echo "$OSTREE_FEATURES" | grep --quiet --no-messages "DevelBuild"; then
+if echo "$OSTREE_FEATURES" | grep --quiet --no-messages "devel"; then
     devel_syms=${G_TEST_SRCDIR}/src/libostree/libostree-devel.sym
 else
     devel_syms=
